executing/executing.py

```diff
@@ -290,11 +318,13 @@ def _for_filename_and_lines(cls, filename, lines):
 
     @classmethod
     def lazycache(cls, frame):
-        if hasattr(linecache, 'lazycache'):
+        # type: (types.FrameType) -> None
+        if sys.version_info >= (3, 5):
             linecache.lazycache(frame.f_code.co_filename, frame.f_globals)
 
     @classmethod
     def executing(cls, frame_or_tb):
+        # type: (types.TracebackType) -> "Executing"
```

Comment:
```suggestion
        # type: (Union[types.TracebackType, types.FrameType]) -> "Executing"
```

Reply:
I was wondering what the other type was here, but hadn't seen any tests for it...